### ISSUE TITLE
Problem when initializing new function 

### ISSUE THREAD

**Issue by arqo123** [kwameopareasiedu/docts#4]
Hello there! Cool package! Hope to bring my impact here soon ;) 

I have a problem when initializing a new function 

**System Info:**
- System: Ubuntu 22.04
- NodeJS: 14.19.1
 

I just wonder if there is a minimum nodejs version to run this project.  If yes, then would be good to mention it in readme and package.json. As for NodeJS 18.14.1 it's working fine.

**Command:** 
`docts init DocumentGenerator`

```
file:///home/arqo/.nvm/versions/node/v14.19.1/lib/node_modules/docts-cli/bin/init.js:2
import { cpSync, existsSync, readdirSync, writeFileSync } from "fs";
         ^^^^^^
SyntaxError: The requested module 'fs' does not provide an export named 'cpSync'
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:124:21)
    at async ModuleJob.run (internal/modules/esm/module_job.js:179:5)
    at async Loader.import (internal/modules/esm/loader.js:178:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5)
    at async handleMainPromise (internal/modules/run_main.js:59:12)
```

**Comment by kwameopareasiedu** [comment]
Hi @arqo123. 

You are right, `cpSync` doesn't exist in Node 14. 

I'll the min version to **Node18** update the README soon

**Comment by arqo123** [comment]
Cool! Thanks! Amazing project btw!  I will let you know if I will find anything more - and I think I've found one more - when initializing the app with `init` command there is a problem with building it.
![image](https://user-images.githubusercontent.com/26055043/220028166-ac7f1b75-0e9a-4377-8446-34618a7d1d82.png)

![image](https://user-images.githubusercontent.com/26055043/220028196-0d00c434-b787-48df-9e5f-c0326fe49613.png)


The one generated with `docts fn new` is working fine.

**Steps to reproduce:**
1. Go to some new folder
2. `docts init <name>`
3. `docts build`

**Comment by kwameopareasiedu** [comment]
@arqo123 This is indeed a bug. The default function should be at `/src/sample/hello/index.ts` instead of `/src/sample/hello/hello.ts`.

Working on an patch. Will update here afterwards...

**Comment by kwameopareasiedu** [comment]
@arqo123 

The bug has been resolved in the [latest release (1.1.1)](https://github.com/kwameopareasiedu/docts/releases/tag/1.1.1) also published on NPM.

Thanks so much for bringing this up

**Comment by arqo123** [comment]
Well done @kwameopareasiedu! Thank You for update! 🎉🍻🚀
